pyproject.toml

```diff
@@ -30,8 +30,10 @@ classifiers = [
     "Programming Language :: Python :: 3.12",
 ]
 dependencies = [
+    "numpy<2.0; python_version == '3.9'",
+    "numpy; python_version == '3.8'",
```

Comment:
```suggestion
    "numpy<2.0; python_version < '3.10'",
```

Reply:
maybe this is easier to read/fix later?

Reply:
I suggest to first drop Python 3.8 and 3.9 and then fix the version requirements for tables and numpy

Reply:
Done, you can merge it :)